model/src/main/scala/za/co/absa/atum/model/utils/OffsetDateTimeSerializer.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.model.dto.utils
+
+import org.json4s.JsonAST.JString
+import org.json4s.{CustomSerializer, JNull}
+
+import java.time.OffsetDateTime
+
+case object OffsetDateTimeSerializer extends CustomSerializer[OffsetDateTime](formats => (
```

Comment:
`formats` is not used
```suggestion
case object OffsetDateTimeSerializer extends CustomSerializer[OffsetDateTime](_ => (
```